airflow/serialization/json_schema.py

```diff
@@ -50,9 +50,9 @@ def iter_errors(self, instance) -> Iterable[jsonschema.exceptions.ValidationErro
         ...
 
 
-def load_dag_schema() -> Validator:
+def load_schema() -> dict:
```

Comment:
```suggestion
def load_dag_schema_json() -> dict:
```

And then leave the "default" one unchanged as `load_dag_schema`

Reply:
Done